### DISCUSSION TITLE
[auth] [ios] Add ability to tag Ente Auth items on iOS

### DISCUSSION THREAD

**Discussion by RossPatterson** [ente-io/ente#2539] [Enhancements]
The desktop Ente Auth app, at least under Windows, supports tagging items by right-clicking on them and choosing "Edit".  There doesn't appear to be a similar function on the iOS mobile app.  Please consider adding one, perhaps through the iPhone's "long press" function.

**Comment by sv3nnie** [comment]
You can. Simply swipe the code from right to left and click 'Edit'.

**Reply by RossPatterson** [reply]
Oh, nice.  😀
